mongodbatlas/database_user.go

```diff
@@ -83,17 +88,17 @@ func resourceDatabaseUserCreate(d *schema.ResourceData, meta interface{}) error
 func resourceDatabaseUserRead(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*ma.Client)
 
-	c, _, err := client.DatabaseUsers.Get(d.Get("group").(string), d.Id())
+	u, _, err := client.DatabaseUsers.Get(d.Get("group").(string), d.Id())
 	if err != nil {
-		return fmt.Errorf("Error reading MongoDB DatabaseUser %s: %s", d.Id(), err)
+		return fmt.Errorf("Error reading MongoDB DatabaseUser %s (%s): %s", d.Id(), d.Get("group").(string), err)
 	}
 
-	d.Set("username", c.Username)
-	d.Set("database", c.DatabaseName)
-	rolesMap := make([]map[string]interface{}, len(c.Roles))
-	for i, r := range c.Roles {
+	d.Set("username", u.Username)
+	d.Set("database", u.DatabaseName)
+	rolesMap := make([]map[string]interface{}, len(u.Roles))
+	for i, r := range u.Roles {
 		rolesMap[i] = map[string]interface{}{
-			"role":       r.RoleName,
+			"name":       r.RoleName,
```

Comment:
I think this was a bug, import wasn't working for roles and I couldn't figure out why until I checked the return value of `d.Set("roles", rolesMap)` which pointed out that `role` was not a valid key.